### PR TITLE
RND-41: Implement endpoint for getting all parking spaces

### PR DESCRIPTION
**Pull request by Nikitosique**
Implement endpoint for getting all parking spaces.

### CODE REVIEW

File: src/main/java/com/devteam/sandboxbasedapp/persistence/entity/ParkingSpaceEntity.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+public class ParkingSpaceEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "status")
+    private String status;
```
Comment (nikitalebed) [inline src/main/java/com/devteam/sandboxbasedapp/persistence/entity/ParkingSpaceEntity.java:24]: Let's create a java enum for the statuses

File: src/main/java/com/devteam/sandboxbasedapp/service/ParkingSpaceService.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.devteam.sandboxbasedapp.service;
+
+import com.devteam.sandboxbasedapp.controller.dto.ParkingSpaceDTO;
+
+import java.util.List;
+
+public interface ParkingSpaceService {
```
Comment (nikitalebed) [inline src/main/java/com/devteam/sandboxbasedapp/service/ParkingSpaceService.java:7]: I don't see any advantages of using this interface. Let's get rid of it and rename TransactionalParkingSpaceService to ParkingSpaceService

File: src/main/java/com/devteam/sandboxbasedapp/persistence/entity/ParkingSpaceEntity.java (hunk 1 above)
Comment (Nikitosique) [reply src/main/java/com/devteam/sandboxbasedapp/persistence/entity/ParkingSpaceEntity.java]: Done.

File: src/main/java/com/devteam/sandboxbasedapp/service/ParkingSpaceService.java (hunk 2 above)
Comment (Nikitosique) [reply src/main/java/com/devteam/sandboxbasedapp/service/ParkingSpaceService.java]: Done.

**Review by nikitalebed — APPROVED** [review]

**Pull request merged**